charts/matrix-stack/configs/matrix-rtc/sfu/ingress-nginx-server-snippet.conf.tpl

```diff
@@ -0,0 +1,18 @@
+{{- /*
+Copyright 2025 New Vector Ltd
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/ -}}
+
+location / {
```

Comment:
This will also match `/sfu/get`. Given longest match wins I think we should add a separate `location /sfu/get` block that does the (probably not-necessary) `proxy_set_header X-Forwarded...` lines so that we don't get the upgrade lines

Reply:
But doesn't the `ingress-controller` already generates `location /sfu/get` due to v ?

```yml
      paths:
      - path: /sfu/get
``` 

I suppose I need to look into what is actually rendered in the pytest env.

Reply:
There's 2 paths on the ingress though `/` and `/sfu/get`. So I'd already expect 2x `location` blocks and for the ingress controller to merge them